tests/lib/rules/jsx-key.js

```diff
@@ -88,5 +90,11 @@ ruleTester.run('jsx-key', rule, {
     options: [{checkFragmentShorthand: true}],
     settings,
     errors: [{message: 'Missing "key" prop for element in array. Shorthand fragment syntax does not support providing keys. Use Act.Frag instead'}]
+  }, {
+    code: '[<App {...obj} key="keyAfterSpread" />];',
+    parser: parsers.BABEL_ESLINT,
+    options: [{checkKeyMustBeforeSpread: true}],
+    settings,
+    errors: [{message: '"key" prop must before "...spread"'}]
```

Comment:
let's include duplicate examples for an HTML element as well. also, duplicate examples for TypeScript, and for eslint's default parser.

Reply:
updates.